.github/workflows/build-ci.yml

```diff
@@ -0,0 +1,39 @@
+name: Paypay Node SDK CI
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      CC_TEST_REPORTER_ID: cbd6f734a0585395a1e664feb0be141affd12ea562966f98fa8f81d29e868e30
+    strategy:
+      matrix:
+        node-version: [14, 12, 10]
```

Comment:
find out the alternative for 
```
-lts/*
-node
```
I am assuming it is the latest LTS and latest version

And please use cache while building
check if anything else is left from travis.yml
https://github.com/paypay/paypayopa-sdk-node/blob/master/.travis.yml

Reply:
Hi @shreyanshp , I have added caching and versions as (ex: 14.x ) which picks 14 related versions. In GitHub Action we don't have option to set **its/***